utils/report_generator.py

```diff
@@ -0,0 +1,197 @@
+import logging
+import os
+from csv import writer, reader
+from datetime import datetime
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+def append_list_as_row(file_name, list_of_elem):
+    """Appends a list of values to an existing or new csv file.
+    
+    Args:
+        file_name (pathlib.PosixPath): The absolute path to the .csv file.
+        list_of_elem (list): a list of elements to write to the .csv file 
+
+    """
+    # Open file in append mode
+    with open(file_name, 'a+', newline='') as write_obj:
+        # Create a writer object from csv module
+        csv_writer = writer(write_obj)
+        # Add contents of list as last row in the csv file
+        csv_writer.writerow(list_of_elem)
+
+
+def initialize_report(file_name):
+    """Initializes a report .csv file with a specified set of headers.
+    
+    Args:
+        file_name (pathlib.PosixPath): The absolute path to the new .csv file 
+
+    Returns:
```

Comment:
I sometimes use the Returns section to say None if it returns None, but then explain what it does to the output if something happens. Or just delete this section if not used.

Reply:
There are a few places this occurs in this script. Maybe clean up the docstrings with the Returns section completed or just left out.

Reply:
added "None" to be consistent with you